### PR TITLE
Added SummaryScreen UI elements and SummaryContent Bloc

### PR DESCRIPTION
**Pull request by veniaming**
demo: https://drive.google.com/open?id=1b5ulChRXnRpn0_Jwvc8DsYFI5XWFIAG1

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -33,14 +34,16 @@ class EndavaProfileApp extends StatelessWidget {
...
         Paths.CONTRIBUTORS: (c) =>
             ContributorsScreen(repository: 'stpopa/endava_profile_app'),
+        Paths.SUMMARY_SCREEN: (c) => SummaryScreen(),
       },
       home: BlocBuilder<AuthenticationBloc, AuthenticationState>(
         builder: (context, state) {
           if (state is AuthenticatedState) {
             return HomeScreen();
           }
 //          if (state is UnauthenticatedState) {
```
Comment (ccosnean) [inline lib/main.dart]: i assume this is leftover of debugging )

File: lib/modules/common/constants.dart [hunk 2]
```diff
@@ -2,4 +2,5 @@ class Paths {
   static const String HOME = '/';
   static const String LOGIN = '/login';
   static const String CONTRIBUTORS = '/contributors';
+  static const String SUMMARY_SCREEN = '/summary_screen';
```
Comment (ccosnean) [inline lib/modules/common/constants.dart:6]: i did refactor the routes in a class... and i'll merge it soon, you'll have to solve some conflicts here.

File: lib/modules/summary/bloc/edit_content_event.dart [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class EditContentEvent {
+  final String text;
+
+  EditContentEvent(this.text);
+
```
Comment (ccosnean) [inline lib/modules/summary/bloc/edit_content_event.dart]: remove spacing

**Review by ccosnean — CHANGES_REQUESTED** [review]

File: lib/main.dart (hunk 1 above)
Comment (veniaming) [reply lib/main.dart]: Removing, my bad).

File: lib/modules/summary/bloc/edit_content_event.dart (hunk 3 above)
Comment (veniaming) [reply lib/modules/summary/bloc/edit_content_event.dart]: done

File: lib/modules/common/constants.dart (hunk 2 above)
Comment (veniaming) [reply lib/modules/common/constants.dart:6]: ok, no probs

**Review by ccosnean — DISMISSED** [review]

**Comment by ccosnean** [comment]
Just merged my PR, Please solve the conflicts, and make sure to use `AppRoute` insted of `PATHS`

**Review by ccosneanu — DISMISSED** [review]

**Review by ccosnean — APPROVED** [review]

**Review by ccosnean — CHANGES_REQUESTED** [review]
Solve conflicts

**Review by ccosnean — DISMISSED** [review]

**Review by ccosnean — DISMISSED** [review]

**Review by ccosnean — APPROVED** [review]

**Pull request merged**